final_prep/annotations.rb

```diff
@@ -2,12 +2,16 @@
 # Try to focus on using correct technical vocabulary.
 # Use the # to create a new comment
 
-# Build a Bear
-
+# Build a Bears
+#create a method name, Build a bear that takes is arguments for the bear's name, age, fur, clothes,  and special_power
 def build_a_bear(name, age, fur, clothes, special_power)
+  #greeting is a string  that interpolates the bear's name
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #demographics is an array that identifies the bear's name and age
   demographics = [name, age]
+  #power_saying is a string that interpolates special_power
   power_saying = "Did you know that I can #{special_power}?"
+#create built_bear hash that has the bear's attributes
```

Comment:
You can still go line by line through this data- of particular importance, where do the values of some of these keys come from?